cmd/argo/commands/lint.go

```diff
@@ -36,30 +38,41 @@ func NewLintCommand() *cobra.Command {
 						return err
 					}
 				}
-				fmt.Printf("%s is valid\n", file)
+				fmt.Printf("%s is valid \u2713\n ", file)
 				return nil
 			}
-
 			for _, file := range args {
 				stat, err := os.Stat(file)
 				errors.CheckError(err)
 				if stat.IsDir() {
-					err := filepath.Walk(file, func(path string, info os.FileInfo, err error) error {
+					filepath.Walk(file, func(path string, info os.FileInfo, err error) error {
 						fileExt := filepath.Ext(info.Name())
 						switch fileExt {
 						case ".yaml", ".yml", ".json":
 						default:
 							return nil
 						}
-						return lint(path)
+						err = lint(path)
+						if err != nil {
+							unvalidWf = fmt.Errorf("unvalid workflow/workflows found")
+							log.Warn(fmt.Errorf("%w \u2717", err))
+						}
+						return nil
 					})
-					errors.CheckError(err)
 				} else {
 					err := lint(file)
-					errors.CheckError(err)
+					if err != nil {
+						unvalidWf = fmt.Errorf("unvalidWf workflow/workflows found")
+						log.Warn(fmt.Errorf("%w \u2717", err))
```

Comment:
Same here. We don't want unicode. You can prepend the error message with the name of the file that caused it.